src/ldp/representation/RepresentationMetadata.ts

```diff
@@ -35,4 +35,14 @@ export interface RepresentationMetadata {
    * Optional timestamp of the representation.
    */
   dateTime?: Date;
+  /**
+   * Optional link type of the representation.
+   * E.g. LINK_TYPE_LDP_C
+   */
+  linkType?: string;
```

Comment:
In theory there can be multiple (not sure what to do with that though).

Reply:
I had already thought of this when I was implementing the InteractionHandler and started working with `includes`, but did not adapt this JSDoc anymore.
I'll combine the feedback from both of you on this topic.
As in the current NSS (HEAD request to a container), the different link types are separated by a comma. In this implementation, is this also okay to see this as one string, each type separated by commas, or do we better store it in an array?

Reply:
They should be combined in a string in the output header, but that should not be done by the ResourceStore, this would be done somewhere closer to where the actual output gets sent to the user. So for the metadata object I would change `linkType` to  
```
 linkRel?: { [id: string]: string }
```
Then the ResourceStore just has to check `metadata.linkRel?.type`.

I don't think there would be multiple values for the same link relationship. We can always change to arrays if I'm wrong.

Reply:
We probably want arrays or sets indeed; because

> the different link types are separated by a comma

- this is just an artifact of syntax (and hence should not be propagated around)
- there are different syntaxes without commas as well